android-quick-start/README.md

```diff
@@ -0,0 +1,88 @@
+# Quickstart: Create an Android app with Amazon Location Maps and Places API
+
+This quickstart app creates a quick start application that demonstrates the following:
+
+1. Adding Amazon Location Maps using Esri as data source to the app.
+2. Adding reverse geocoding using Amazon Location Places API to demonstrate a place picker.
+
+After a successful build the app will look like:
+
+![](screenshot.png)
+
+## Prerequisities
+1. A Amazon Location map resource
+1. A Amazon Location place-index resource
+1. A Cognito identity pool for unauthorized access
+
+## How to set up the project
+Before the project can be built we need the following steps to be completed:
+1. Configuring amplify to set map, place-index, and cognito identity-pool id.
+1. Setting up Android SDK path using `local.properties` file or opening the project in Android Studio.
+
+### Configuring Amplify
+This project uses a JSON configuration following Amplify's configuration schema in `app/res/raw/amplifyconfiguration.json`. To use the quick start, we need to replace the following in the configuration JSON.
+1. [region]
+1. [map name]
+1. [map style name]
+1. [place index name]
+
+For example, if we are building this application for region `us-west-2`, map resource name `sample-map`, map style name `VectorEsriStreets`, place-index resource name `sample-place-index`, and cognito pool id `us-west-2:cognito-pool-id`, `amplifyconfiguration.json` will look like the following:
```

Comment:
For example, if we are building this application for region `us-west-2`, map resource name `sample-map`, map style name `VectorEsriStreets`, place-index resource name `sample-place-index`, and Identity Pool id `us-west-2:cognito-pool-id`, the `amplifyconfiguration.json` will look like the following:

Reply:
I also made this change.